packages/core/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@bundler/core",
+  "version": "1.0.0",
+  "description": "bundler core",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "scripts": {
+    "test:unit": "jest --config=./tests/configurations/unit/jest.config.js",
+    "format": "prettier --check . --ignore-path ../../.prettierignore",
+    "format:fix": "prettier --write . --ignore-path ../../.prettierignore",
+    "prelint:fix": "npm run format:fix",
+    "prelint": "npm run format",
+    "lint": "eslint .",
+    "lint:fix": "eslint --fix .",
+    "release": "standard-version",
+    "test": "npm run test:unit",
+    "prebuild": "npm run clean",
+    "build": "tsc --project tsconfig.build.json",
+    "start": "npm run build && cd dist && node ./index.js",
+    "clean": "rimraf dist",
+    "prepack": "npm run build"
+  },
+  "repository": {
+    "type": "git",
+    "url": ""
+  },
+  "author": "MapColonies",
+  "license": "ISC",
+  "files": [
+    "dist/**/*"
+  ],
+  "dependencies": {
+    "@bundler/child-process": "*",
+    "@bundler/common": "*",
+    "@bundler/github": "*",
```

Comment:
once this pr is approved we need to run `npx lerna version major --conventional-commits`, this will:
1. set a fixed version on the `lerna.json` to be `1.0.0`
2. set the version of each of the packages to `1.0.0`. on each of the package's `package.json`
3. set the inner dependencies to version `"1.0.0"` instead of `"*"`
4. generate a `CHANGELOG` for each of the packages
5. github push and release v1.0.0